### PR TITLE
When launching app window on macOS, make application a foreground app…

### LINKED ISSUE #109: OSX keyboard event input.
When running on OSX, it turns out that keyboard events don't get sent to systray when running as `systray.RunWithAppWindow`.

To fix this, add the following at the start of configureAppWindow() in systray_browser_darwin.m

```
  ProcessSerialNumber psn = {0, kCurrentProcess};
  OSStatus status = TransformProcessType(&psn, kProcessTransformToForegroundApplication);
```

### PR DESCRIPTION
**Pull request by oxtoacart**
…lication, closes #109

### CODE REVIEW

File: example/main.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func onReady() {
 				mEnabled.SetTitle("Disabled")
 				mEnabled.Disable()
 			case <-mUrl.ClickedCh:
-				systray.ShowAppWindow("https://www.getlantern.org")
+				systray.ShowAppWindow("https://www.github.com/getlantern/lantern")
```
Comment (oxtoacart) [inline example/main.go]: Has more interesting UI controls with which to test.

File: systray_darwin.m [hunk 2]
```diff
@@ -187,8 +187,6 @@ - (void) quit
 
 @end
 
-void configureAppWindow(char* title, int width, int height);
```
Comment (oxtoacart) [inline systray_darwin.m]: This was already declared in the header, didn't need to re-declare it here.

**Comment by atavism** [comment]
LGTM!

**Comment by oxtoacart** [comment]
Thanks @atavism!

**Pull request merged**